docs/introduction.md

```diff
@@ -59,3 +59,9 @@ After the promise is settled, a second action will be dispatched. If the the pro
   }
 }
 ```
+
+The default promise status suffix can be imported from this module.
+
+```js
+import { PENDING, FULFILLED, REJECTED } from 'redux-promise-middleware'
```

Comment:
Oh right, thanks. I think I'd prefer `PENDING` instead of `DEFAULT_PENDING` but I get naming conflicts with lines [30](https://github.com/pburtchaell/redux-promise-middleware/blob/master/src/index.js#L30) to [32](https://github.com/pburtchaell/redux-promise-middleware/blob/master/src/index.js#L32).

Should I prefix them with eg. `SUF_` or should we keep the exported constants with `DEFAULT_`?